app/src/main/java/io/qtalk/qgamejsinterfacetester/JSInterface.kt

```diff
@@ -0,0 +1,71 @@
+package io.qtalk.qgamejsinterfacetester
+
+import android.webkit.JavascriptInterface
+
+@Suppress("unused")
+class JSInterface(private var jsInterfaceBridge: JSInterfaceBridge? = null) {
+
+
+    fun setJSInterfaceBridge(jsInterfaceBridge: JSInterfaceBridge?){
+        this.jsInterfaceBridge = jsInterfaceBridge
+    }
+
+    @JavascriptInterface
+    fun getUserAuthToken(): String {
+        return jsInterfaceBridge?.getUserAuthToken()?:""
```

Comment:
We dont wish to work without an auth token, would be good to put a `!!` here or throw an exception

Reply:
done